game/state/shared/aequipment.cpp

```diff
@@ -730,42 +737,51 @@ void AEquipment::fire(GameState &state, Vec3<float> targetPosition, StateRef<Bat
 			return;
 		}
 		// Throw item (accuracy applied inside)
-		item->throwItem(state, targetPosition, velocityXY, velocityZ, true);
+
+		for (number_of_shots; number_of_shots > 0; number_of_shots--)
```

Comment:
Same unnecessary statement in for loop as above

Reply:
Change to?

```for (int shot_number = 0; shot_number < number_of_shots; shot_number++)```